python/metatensor-torch/tests/atomistic/openmm_force.py

```diff
@@ -0,0 +1,186 @@
+import os
+
+import ase.io
+import metatensor_lj_test
+import numpy as np
+import pytest
+from ase.build import bulk
+
+from metatensor.torch.atomistic.ase_calculator import MetatensorCalculator
+from metatensor.torch.atomistic.openmm_force import get_metatensor_force
+
+
+try:
+    import NNPOps  # noqa: F401
+    import openmm
+    import openmmtorch  # noqa: F401
+
+    HAS_OPENMM = True
+except ImportError:
+    HAS_OPENMM = False
```

Comment:
Since we want to skip the whole file if one of the dependency is missing, we should use importskip:

```suggestion
openmm = pytest.importskip("openmm")

pytest.importskip("NNPOps")
pytest.importskip("openmmtorch")
```